android/src/test/java/com/instabug/flutter/InstabugApiTest.java

```diff
@@ -345,6 +347,32 @@ public void testClearAllExperiments() {
         mInstabug.verify(Instabug::clearAllExperiments);
     }
 
+    @Test
+    public void testAddFeatureFlags() {
+       Map<String,String > featureFlags = new HashMap<>();
+        featureFlags.put("key1","variant1");
+        api.addFeatureFlags(featureFlags);
+        List<IBGFeatureFlag> s=new ArrayList<IBGFeatureFlag>();
+        s.add(new IBGFeatureFlag("key1","variant1"));
+        mInstabug.verify(() -> Instabug.addFeatureFlags(s));
+    }
+
+    @Test
+    public void testRemoveFeatureFlags() {
+        List<String> featureFlags = Arrays.asList("premium", "star");
+
+        api.removeFeatureFlags(featureFlags);
+
+        mInstabug.verify(() -> Instabug.removeFeatureFlag(featureFlags));
+    }
+
+    @Test
+    public void testClearAllFeatureFlags() {
+        api.removeAllFeatureFlags();
+
+        mInstabug.verify(Instabug::removeAllFeatureFlags);
```

Comment:
Can we review the featureFlags content to confirm the removal?

Reply:
There is no way to export feature tags from the native SDK to verify whether feature tags have been removed or added